arm-sql/2015-05-01-preview/examples/ServerCreateMax.json

```diff
@@ -27,14 +27,11 @@
         "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/sqlcrudtest-7398/providers/Microsoft.Sql/servers/sqlcrudtest-4645",
         "name": "sqlcrudtest-4645",
         "type": "Microsoft.Sql/servers",
-        "location": "Japan East",
```

Comment:
I think the original examples should still be kept in 2014-04-01 folder, and the new examples should be new files in 2015-01-01-preview folder. Swagger folks to confirm.